abb_irb4600_support/urdf/irb4600_20_250_macro.xacro

```diff
@@ -0,0 +1,175 @@
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_2.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_3">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_3.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_3.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_4">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_4.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_4.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_5">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_5.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_5.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}link_6">
+      <collision name="collision">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/collision/link_6.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+      </collision>
+      <visual name="visual">
+        <geometry>
+          <mesh filename="package://abb_irb4600_support/meshes/irb4600_20_250/visual/link_6.stl"/>
+        </geometry>
+        <xacro:material_abb_white/>
+        <material name="yellow"/>
+      </visual>
+    </link>
+    <link name="${prefix}tool0"/>
+    <!-- end of link list -->
+
+    <!-- joint list -->
+    <joint type="revolute" name="${prefix}joint_1">
+      <origin xyz="0 0 0.495" rpy="0 0 0"/>
+      <axis xyz="0 0 1"/>
+      <parent link="${prefix}base_link"/>
+      <child link="${prefix}link_1"/>
+      <limit effort="0" lower="-3.141" upper="3.141" velocity="3.054"/>
```

Comment:
Please use `${radians(..)}` to convert from degrees to radians. You can use the specsheet to find the joint limits in radians (which you probably also used to convert things manually here).